indra/newview/skins/default/xui/en/floater_marketplace.xml

```diff
@@ -169,7 +169,7 @@
         follows="all"
         layout="topleft"
         left="0"
-        name="marketplace_contents"
+        name="webbrowser"
```

Comment:
Two other places are also using marketplace_contents, in llviewermedia.cpp and llviewerwindow.cpp

Reply:
Yes, sorry, it's there in develop, but was removed in 2025.08